0028-web-monetization/0028-web-monetization.md

```diff
@@ -41,33 +41,37 @@ Web Monetization makes use of [SPSP](../0009-simple-payment-setup-protocol/0009-
 
 This flow refers to the user's **browser** and the user's **provider**, [defined above](#terminology).
 
-- The user navigates their browser to a webpage.
-- The browser sets `document.monetization` to an Object which implements [EventTarget](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget).
-- The browser sets `document.monetization.state` to `pending`.
-- The browser observes the `<head>` and looks for a Web Monetization `<meta name="monetization">` tag ([specified below](#meta-tags)).
+- The user's agent navigates a webpage.
```

Comment:
I changed these to consistently use `user's agent` though perhaps it should have been the other way around?